lib/ansible/modules/cloud/ovirt/ovirt_clusters.py

```diff
@@ -208,6 +208,9 @@
         description:
             - "The compatibility version of the cluster. All hosts in this
                cluster must support at least this compatibility version."
+    mac_pool:
+        description:
+            - "MAC pool to be used by this cluster."
```

Comment:
No, I didn't mean to write the 'If user..' :D Just leave a note about i'ts supported since 4.1